container/state.go

```diff
@@ -59,10 +60,7 @@ func (s StateStatus) Err() error {
 
 // NewState creates a default state object with a fresh channel for state changes.
```

Comment:
Is the "with a fresh channel for state changes." bit still correct now?

(just a quick glance over the changes - haven't fully reviewed yet)

Reply:
Nope, it's outdated. Fixed, thanks!